modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx

```diff
@@ -12,12 +12,25 @@
  * details.
  */
 
+import ClayButton from '@clayui/button';
+import {useModal} from '@clayui/modal';
 import React from 'react';
 
 import BasePage from '../../components/BasePage';
+import Modal from '../../components/CreatePropertyModal';
 
 const Step = () => {
-	return <BasePage title={Liferay.Language.get('property')}></BasePage>;
+	const {observer, onOpenChange, open} = useModal();
+
+	return (
+		<>
+			<BasePage title={Liferay.Language.get('property')}></BasePage>;
```

Comment:
Since there's no content in the component, you can write using self-closing:
`<BasePage title={Liferay.Language.get('property')} />`

Reply:
Done.